src/config.rs

```diff
@@ -242,3 +251,58 @@ fn get_global_config_path() -> Option<PathBuf> {
 fn get_global_config_path() -> Option<PathBuf> {
     Some(PathBuf::from("/etc/rustic"))
 }
+
+/// A command to be called which can be given as CLI option as well as in config files
+/// `CommandInput` implements Serialize/Deserialize as well as FromStr.
+
+// Note: we use CommandInputInternal here which itself impls FromStr in order to use serde_as PickFirst for CommandInput.
+#[serde_as]
+#[derive(Debug, Default, Clone, Serialize, Deserialize, Merge)]
+pub struct CommandInput(#[serde_as(as = "PickFirst<(_,DisplayFromStr)>")] CommandInputInternal);
```

Comment:
If we want to backport this dual compatibility (`String` and `Vec<String>`) for commands in `rustic_core` crate (regarding the changes we made a few weeks ago concerning `password` commands, I assume this `CommandInput` code might be moved to `rustic_core` crate is that right ?

Reply:
Exactly.

Reply:
This PR is now based on https://github.com/rustic-rs/rustic_core/pull/252